### PR TITLE
{bio}[intel/2018b] nanopolish v0.10.2 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/712166437c73fe258659e17d00432fb7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2452.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cc90c015381cc463bf6092781f9c1ace for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
node2094.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/741e4e68141552043a84e1845942a617 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5ec921a0d896033514a3e18a19110a84 for a full test report.

File: easybuild/easyconfigs/h/HTSlib/HTSlib-1.9-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# 1.4 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HTSlib/HTSlib-1.9-intel-2018b.eb]: @boegel isn't this already included in a slightly different form in #7322 ?

Comment (boegel) [reply easybuild/easyconfigs/h/HTSlib/HTSlib-1.9-intel-2018b.eb]: Yes it is, let's get that one merged first...

**Comment by boegel** [comment]
@verdurin updated after #7322 was merged

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2444.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fbd7533650671fb0a763a32fb36eab09 for a full test report.

**Comment by boegel** [comment]
(close/open to wake up Travis)

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**